charmcraft.yaml

```diff
@@ -12,15 +12,19 @@ parts:
   charm:
     build-packages:
       - git
+      - rustc
+      - cargo
     charm-binary-python-packages:
+      - pip>=24
       - jsonschema
       - cryptography
       - pyyaml
       - requests
       - ops
       - wheel==0.37.1
       - setuptools==45.2.0
-      - pydantic>=2
+      - pydantic
+      - pydantic-core
```

Comment:
Should be bring back the `>=2` pin?

Reply:
Those binary packages are not doing much anyway, for some reason `charcmraft` still builds them from scratch; my current guess is that `PYDEPS` are always rebuilt